template/templates/views/edit_feed.html

```diff
@@ -87,7 +87,10 @@ <h3>{{ t "page.edit_feed.last_parsing_error" }}</h3>
         <label><input type="checkbox" name="fetch_via_proxy" value="1" {{ if .form.FetchViaProxy }}checked{{ end }}> {{ t "form.feed.label.fetch_via_proxy" }}</label>
         {{ end }}
         <label><input type="checkbox" name="disabled" value="1" {{ if .form.Disabled }}checked{{ end }}> {{ t "form.feed.label.disabled" }}</label>
-        <label><input type="checkbox" name="hide_globally" value="1" {{ if .form.HideGlobally }}checked{{ end }}> {{ t "form.feed.label.hide_globally" }}</label>
+        
+        {{ if not .form.CategoryHidden }}
```

Comment:
it might make sense to set the checkbox to disabled+checked instead, and maybe add a comment `(from category)` to clarify why that is :thinking:

Reply:
You're right. I just tried to avoid new translations. 😅